app/views/programme_types/_intention.html.erb

```diff
@@ -0,0 +1,7 @@
+<% if programme_type.activity_type_todos.none? %>
+  <%= render IconComponent.new(name: 'chalkboard-user') %> <%= t('programme_types.intention.adults') %>
```

Comment:
Could use users or maybe people-group https://fontawesome.com/icons/people-group?f=classic&s=solid if you switch to using a school icon.

I think the pupil icon works

Reply:
Thanks for these suggestions, have made the changes